capstone/src/main/webapp/findStudySet.html

```diff
@@ -1,27 +1,56 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="UTF-8">
-    <title>Studentia-Find Study Set</title>
+    <title>Studentia-Homepage</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <!-- Compiled and minified CSS -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
     <!--Import Google Icon Font-->
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    <!--Import materialize.css-->
-    <link type="text/css" rel="stylesheet" href="css/materialize.min.css"  media="screen,projection"/>
-
     <!--Let browser know website is optimized for mobile-->
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
   </head>
-  <body>
-     <h3>Find Study Set</h3>
+  
+  <body onload="most_recent_studysets()">
+
+    <nav class="nav-wrapper">
+        <a href="#" class="brand-logo">Studentia</a>
+        <a href="#" class="sidenav-trigger" data-target="slide_out" ><i class="material-icons">menu</i></a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+          <li><a href="findStudySet.html">Search</a></li>
+          <li><a href="index.html">Home</a></li>
+        </ul>
+    </nav>
+
+    <ul class="sidenav" id="slide_out">
+      <li><a href="findStudySet.html">Search</a></li>
+      <li><a href="index.html">Home</a></li>
+    </ul>
+  
+    <div class="row">
```

Comment:
Remove the search bar from this PR. None of the backend functionality works yet so it just causes confusion. The goal for your master branch is that you can deploy the code from master to production at any time. So you don't want to submit anything that looks half-finished to the user.